src/Basque/Mews.Fiscalizations.Basque/Dto/Bizkaia/Header/BizkaiaData.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Mews.Fiscalizations.Basque.Dto.Bizkaia.Header;
+
+public partial class BizkaiaData
```

Comment:
Isn't this supposed to be provided in the header?

If yes, then BizkaiaHeaderData makes more sense?